A66-otel-stats.md

```diff
@@ -406,22 +407,22 @@ the data.
 *   **grpc.client.attempt.started** <br>
     The total number of RPC attempts started, including those that have not completed. <br>
     *Attributes*: grpc.method, grpc.target <br>
-    *Type*: Counter <br>
+    *Type*: Counter (integer) <br>
```

Comment:
I think we might want to say something more precise than just "integer" vs. "floating point".  For example, would it be okay for a gRPC implementation to use a 32-bit integer here?  I assume not, in which case we need language here saying so.